scripts/test-check-packages.sh

```diff
@@ -98,39 +140,33 @@ for d in test/packages/${PACKAGE_TEST_TYPE:-other}/${PACKAGE_UNDER_TEST:-*}/; do
   package_to_test=$(basename "${d}")
 
   if [ "${PACKAGE_TEST_TYPE:-other}" == "benchmarks" ]; then
-    # It is not used PACKAGE_UNDER_TEST, so all benchmark packages are run in the same loop
-    if [ "${package_to_test}" == "pipeline_benchmark" ]; then
-      rm -rf "${PWD}/build/benchmark-results"
-      elastic-package benchmark pipeline -C "$d" -v --report-format xUnit --report-output file --fail-on-missing
+    # FIXME: There are other packages in test/packages/benchmarks folder that are not tested like rally_benchmark
```

Comment:
`rally_benchmark` package is located in `test/packages/benchmarks` folder but it is not tested at all. 
I don't know if this could be tested as it is here. IIUC according to the docs it is required also `esrally` tool.